docs/details/blocks/General/Sub.md

```diff
@@ -0,0 +1,11 @@
+In Chainblocks a block receives an input, processes it, and produces an output. Usually this output is different from the input that the block processed. 
+
+However, in some cases, we want the block to return the same value as its output that it reveived as an input (for example when we want the next block in the sequence to process the same input that the previous block processed).
```

Comment:
this is wrong, works on multiple blocks, the definition should be in the line of being a flow bypass
![image](https://user-images.githubusercontent.com/7008900/165871679-71cf485b-4944-4582-8813-63ac203fed0c.png)

Where you can have multiple blocks.
Also every variable assigned inside Sub will be available outside as well as Sub will always be executed. I suggest trying to use the block @satendrapalsingh

Reply:
I see you added mention to sequence under ok. but maybe give more emphasis still of the utility of this.

Reply:
I have rewritten a few paras to 
> highlight earlier in the text that this works for a block sequence too
> emphasise that the variables inside `Sub` can be accessed outside it

Will cover this topic in more detail under `program control flow` that is planned as part of the `Chainblocks Primer` doc.